src/components/Graph/gl/GraphBolusGl.js

```diff
@@ -262,19 +267,24 @@ class GraphBolusGl extends GraphRenderLayerGl {
     const y = this.yAxisBottomOfBolus;
     const centerY = y - height / 2;
 
-    const geometry = GraphShapeGeometryFactory.makeRectangleGeometry({
-      width: width - 1,
-      height: height - 1,
-    });
-    const mesh = new THREE.Mesh(geometry, this.bolusRectMaterial);
-    this.addAutoScrollableObjectToScene(this.scene, mesh, {
-      x: x + 0.5,
-      y: centerY,
-      z,
-      contentOffsetX: this.contentOffsetX,
-      pixelsPerSecond: this.pixelsPerSecond,
-      shouldScrollX: true,
-    });
+    const rectangleGeometryWidth = width - 1;
+    const rectangleGeometryHeight = height - 1;
+    console.log(`rendering bolus rect, width: ${width}, height: ${height}`);
```

Comment:
Should this console log be commented out or removed?  I'm assuming any logs intentionally left would be called via `Logger`

Reply:
Yes, sorry. Forgot to eslint it (which would have caught it). (Hmmm.. thought I had eslint enabled in Travis, will look.)